version.properties

```diff
@@ -1,5 +1,5 @@
 #Currently building Mockito version
-version=2.25.5
+version=2.26.0
```

Comment:
I don't think this warrants a new minor release. I think this should be just 2.25.5

Reply:
No probs. I generally like to publish new minor every few weeks. It's arbitrary what we warrant a new minor release, and that's OK.

I need to finally set up the automation that automatically pushes to MavenCentral if the version is next minor.